src/DGtal/images/ImageContainerBySTLVector.h

```diff
@@ -129,6 +129,7 @@ namespace DGtal
   public:
 
     typedef ImageContainerBySTLVector<TDomain, TValue> Self;
+    typedef std::vector<TValue> Parent;
```

Comment:
Should we set an identical typedef ... Container ?

Reply:
Added